modules/wri_node/config/install/field.storage.node.field_show_more_link.yml

```diff
@@ -0,0 +1,18 @@
+uuid: 4e19f02b-cd7a-44bd-807f-968fd5cf6d93
```

Comment:
@komejo I am not able to run the updates and I think it is because this config is not being imported. 

Here is the error:

```
 ---------------- ----------- --------------- ---------------------------------
  Module           Update ID   Type            Description
 ---------------- ----------- --------------- ---------------------------------
  wri_io_content   9301        hook_update_n   9301 - Add the 'Show More Link'
                                               field to the International
                                               Office content type.
  wri_program      9301        hook_update_n   9301 - Add the 'Show More Link'
                                               field to the Program Center
                                               content type.
  wri_project      9301        hook_update_n   9301 - Add the 'Show More Link'
                                               field to the Project Detail
                                               content type.
 ---------------- ----------- --------------- ---------------------------------

 Do you wish to run the specified pending updates? (yes/no) [yes]:
 >

>  [notice] Update started: wri_project_update_9301
>  [error]  Attempt to create a field field_show_more_link that does not exist on entity type node.
>  [error]  Update failed: wri_project_update_9301
>  [notice] Update started: wri_program_update_9301
>  [error]  Attempt to create a field field_show_more_link that does not exist on entity type node.
>  [error]  Update failed: wri_program_update_9301
>  [notice] Update started: wri_io_content_update_9301
>  [error]  Attempt to create a field field_show_more_link that does not exist on entity type node.
>  [error]  Update failed: wri_io_content_update_9301
 [error]  Update aborted by: wri_project_update_9301, wri_program_update_9301, wri_io_content_update_9301
```

Reply:
Forgot to add the field.storage configs! Added it to `wri_io_content`, and made that a dependency for the other two updates. I also added the configs to update the form display, but it doesn't look like `installConfig` or `updateConfig` work when there are existing configs, so we might have to manually update the form display for these three content types to move the '	Show More Experts Link' under the 'Featured Experts' field. (Unless you know a trick for getting the existing form display config to update?)

Reply:
Ah! The trick is to use `installConfig`, with `, 'install', 'TRUE'` to replace existing configs. Should work as expected now.

Reply:
@komejo - hmmm. I am still getting that error, on the multidev, were you able to get it work locally?

```
 ---------------- ----------- --------------- ---------------------------------
  Module           Update ID   Type            Description
 ---------------- ----------- --------------- ---------------------------------
  wri_io_content   9301        hook_update_n   9301 - Add the 'Show More Link'
                                               field to the International
                                               Office content type.
  wri_program      9301        hook_update_n   9301 - Add the 'Show More Link'
                                               field to the Program Center
                                               content type.
  wri_project      9301        hook_update_n   9301 - Add the 'Show More Link'
                                               field to the Project Detail
                                               content type.
 ---------------- ----------- --------------- ---------------------------------

 Do you wish to run the specified pending updates? (yes/no) [yes]:
 >

>  [notice] Update started: wri_project_update_9301
>  [error]  Attempt to create a field field_show_more_link that does not exist on entity type node.
>  [error]  Update failed: wri_project_update_9301
>  [notice] Update started: wri_program_update_9301
>  [error]  Attempt to create a field field_show_more_link that does not exist on entity type node.
>  [error]  Update failed: wri_program_update_9301
>  [notice] Update started: wri_io_content_update_9301
>  [error]  Attempt to create a field field_show_more_link that does not exist on entity type node.
>  [error]  Update failed: wri_io_content_update_9301
 [error]  Update aborted by: wri_project_update_9301, wri_program_update_9301, wri_io_content_update_9301
 [error]  Finished performing updates.
```

Reply:
@aprice42 Forgot to update the multidev! Apologies, it's updating now.